tests/test_onnxml_imputer_converter.py

```diff
@@ -21,7 +21,7 @@
 class TestONNXImputer(unittest.TestCase):
     def _test_imputer_converter(self, norm):
         warnings.filterwarnings("ignore")
-        model = SimpleImputer(strategy="mean", fill_value="nan")
+        model = SimpleImputer(strategy="constant")
```

Comment:
Does this mean that onnx imputation supports only constant? It looks then that they somehow support both mean and constants right? Should we have different tests with different modalities for the `SimpleImputer`?

Reply:
Yes it seems so (https://github.com/onnx/onnx/blob/master/docs/Operators-ml.md#aionnxmlimputer)

Added a test case for all cases where they don't throw unsupported (https://github.com/onnx/sklearn-onnx/blob/master/skl2onnx/operator_converters/imputer_op.py)